test/logger_test.rb

```diff
@@ -242,6 +242,16 @@
       assert_equal 'StandardError', actual_log[:error][:type]
       assert_equal 'Unknown error', actual_log[:error][:message]
     end
+
+    it 'should log error type properly even when active_support/json is required' do
+      require 'active_support/json'
```

Comment:
`active_support` is already a development dependency in twiglet because of `simplycop` which required that gem.